packages/cli/src/options/bundlerOptions/network.ts

```diff
@@ -6,13 +6,15 @@ export interface INetworkArgs {
   "p2p.host": string;
   "p2p.port": number;
   "p2p.bootEnrs": string[];
+  "p2p.dataDir": string;
 }
 
 export function parseArgs(args: INetworkArgs): IBundlerOptions["p2p"] {
   return {
     host: args["p2p.host"],
     port: args["p2p.port"],
     bootEnrs: args["p2p.bootEnrs"],
+    dataDir: args["p2p.dataDir"],

```

Comment:
the dataDir should be the same for both. We don't need a separate configuration setting for this. Please re-use the earlier dataDir parameter.

Reply:
I'll remove it later, I need this temporarily for testing